shell/platform/darwin/ios/ios_external_texture_gl.mm

```diff
@@ -58,8 +61,9 @@
                                  CVOpenGLESTextureGetName(texture_ref_), GL_RGBA8_OES};
   GrBackendTexture backendTexture(bounds.width(), bounds.height(), GrMipMapped::kNo, textureInfo);
   sk_sp<SkImage> image =
-      SkImage::MakeFromTexture(canvas.getGrContext(), backendTexture, kTopLeft_GrSurfaceOrigin,
+      SkImage::MakeFromTexture(context, backendTexture, kTopLeft_GrSurfaceOrigin,
                                kRGBA_8888_SkColorType, kPremul_SkAlphaType, nullptr);
+  FML_CHECK(image) << "Failed to create SkImage from Texture.";
```

Comment:
This will cause a crash if we fail to create the image.  I think that's desired, but not sure.  If it is, we should get rid of the `if (image)` below.  Otherwise, this should be `FML_DCHECK`.

Reply:
@dnfield , i'm worried about crashing for unexpected reasons, so I changed it to DCHECK to maintain the same semantics as before. If we decide to make the process terminate, we can do it as a later commit.